Applications/CLI/ogs.cpp

```diff
@@ -57,7 +57,9 @@ int main(int argc, char* argv[])
         "See accompanying file LICENSE.txt or "
         "http://www.opengeosys.org/project/license\n"
         "version: " +
-            BaseLib::BuildInfo::git_describe,
+            BaseLib::BuildInfo::git_describe + "\n" +
+        "CMake arguments: " +
+            BaseLib::BuildInfo::cmake_args,
```

Comment:
As far as I understand, in the past some user/developer reported/experienced some problems caused by manually set cmake variables, which now are reported by `ogs --help`. That looks like nice feature.

Maybe also some people have problems caused by "dirty" source directories. Therefore (different PR) it might also be helpful to write a status message like "Warning: Source dir dirty, there were changes in the following files: ..." to the screen (and maybe also to each vtu file that we write).

Reply:
the '-dirty' postfix is attached by [git describe](https://git-scm.com/docs/git-describe#git-describe---dirtyltmarkgt) in that case...

Reply:
Ah, OK.